scss/_labels.scss

```diff
@@ -43,10 +43,9 @@ a.label {
 .label-pill {
   padding-right: .6em;
   padding-left: .6em;
-  border-radius: 1rem;
+  @include border-radius(1rem);
 }
 
-
```

Comment:
Extra space was removed because it's not there between the `.label-pill` and `a.label` so it's inconsistent to have the double newline here